src/main/java/frc/robot/commands/CDSForwardCommand.java

```diff
@@ -4,23 +4,31 @@
 
 package frc.robot.commands;
 
+import org.opencv.features2d.FastFeatureDetector;
+
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.CDSSubsystem; 
+import frc.robot.subsystems.CDSSubsystem;
+import frc.robot.subsystems.ShooterSubsystem; 
 
 public class CDSForwardCommand extends CommandBase {
   /** Creates a new IntakeForwardCommand. */
   private final CDSSubsystem mCDSSubsystem;
+  private final ShooterSubsystem mShooterSubsystem;
   
-  public CDSForwardCommand(CDSSubsystem CDSSubsystem) {
+  public CDSForwardCommand(CDSSubsystem CDSSubsystem, ShooterSubsystem shooterSubsystem) {
     //Use addRequirements() here to declare subsystem dependencies.
     addRequirements(CDSSubsystem); 
+    addRequirements(shooterSubsystem);
+    mShooterSubsystem = shooterSubsystem;
     mCDSSubsystem = CDSSubsystem;
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
     mCDSSubsystem.CDSBeltWheelControllerToggle(false);
+    mShooterSubsystem.runCargo(true, false);
```

Comment:
Fine for now while we're testing, but may want a nicer handoff between belt and cargo stopper

Reply:
True, I hate the method call myself (if that's what you mean by handoff) but I can't get boolean... to work in attributes.
If you meant a second command or a delay, I think sequencing that might be a bit too hard, because the stopper is only running for the unpredictable chance a second ball pushes the top ball through the stopper.